server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -29,10 +37,23 @@ namespace irods {
             json plug{
                 {"name", itr->c_str()},
                 {"type", _type_name},
-                {"version", ""},
-                {"checksum_sha256", ""}
+                {"version", ""}
```

Comment:
Is `version` ever updated?

Reply:
At the moment the `version` is never updated.

Reply:
Ok. Please capture that in an issue.